src/Tools/CoreFx.Tools/GenerateBindingRedirect.cs

```diff
@@ -94,14 +94,16 @@ public override bool Execute()
                 {
                     XElement codeBase = new XElement(ns + "codeBase",
                                                 new XAttribute("version", result.Version),
-                                                new XAttribute("href", Path.GetFullPath(assembly.ItemSpec)));
+                                                new XAttribute("href", Path.GetFileName(assembly.ItemSpec)));
```

Comment:
Yes, since devmode is set to true, it will look for bind all the assemblies from DEVPATH.